LICENSE

```diff
@@ -1,6 +1,7 @@
 MIT License
 
 Copyright (c) 2016 David Yahalomi
+Copyright (c) 2020 Felix Mannhardt
```

Comment:
@fmannhardt let's maybe keep both of you? I mean it needs to be changed anyway, to Horia Miron and probably add you as well.